opdracht-3.js

```diff
@@ -0,0 +1,28 @@
+import {inventory} from "./main.js";
+
+/*const Brands = inventory.map((tvs) =>{
+    return tvs.brand
+})
+console.log(Brands)
+
+const tvBrands = Brands.filter((a, b) => {
+    return Brands.indexOf(a) === b;
+})
+console.log(tvBrands)*/
+
+
+
+function brandsWeSell() {
+    const Brands = inventory.map((tvs) => {
```

Comment:
Variabelnamen camelCase schrijven
```suggestion
    const brands = inventory.map((tvs) => {
```